### ISSUE TITLE
Is this prop type necessary?

### ISSUE THREAD

**Issue by zefexdeveloper** [lfades/next-with-apollo#34]
I was wondering why there's the `apollo` in the `PropTypes` as it doesn't receive `apollo` as props, only `apolloState` and `appProps`, am I missing something here?

https://github.com/lfades/next-with-apollo/blob/f6530cf8ad73836953ae9693c1f38ab260b63158/src/withApollo.tsx#L41

**Comment by lfades** [comment]
Yes, it's being used [here](https://github.com/lfades/next-with-apollo/blob/f6530cf8ad73836953ae9693c1f38ab260b63158/src/withApollo.tsx#L105) and [here](https://github.com/lfades/next-with-apollo/blob/f6530cf8ad73836953ae9693c1f38ab260b63158/src/withApollo.tsx#L110), it's also required by the `ApolloProvider` in `_app.js`
